sumologic/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Wait for supervisor to start
+echo "Waiting for supervisor to start"
+wait_listen tcp 9632 30
+
+# Ensure environment is clean for service testing
+source ./plan.sh
+hab svc unload "${HAB_ORIGIN}/${pkg_name}"
+
+# Build and install
+set -e
+build
+source results/last_build.env
+hab pkg install "results/${pkg_artifact}"
+hab svc load "${pkg_ident}"
```

Comment:
is it possible to `hab pkg exec {pkg_ident} sumologic --help`? Please note, I don't know what binaries are created.  So perhaps could at least validate binaries will run without seg faults etc.

Reply:
I agree with this sentiment. But I disagree with the current implementation. You're currently testing the java service wrapper by calling 'wrapper --help'. While the wrapper is used for sumologic, its not sumologic itself. (correct me if I am wrong here).

I think the more generic question here is: Can we call part of sumologic in a test to ensure that its build/binary is sane?